beets/ui/commands.py

```diff
@@ -1017,6 +1018,12 @@ def import_func(lib, opts, args):
     metavar='ID',
     help=u'restrict matching to a specific metadata backend ID'
 )
+import_cmd.parser.add_option(
+    u'--set-field', dest='set_fields', action='callback',
```

Comment:
Agreed, and there's the `metavar` value which makes it sufficiently explicit. I'll change this.